qan-api2/models/metrics.go

```diff
@@ -1026,3 +1026,102 @@ func (m *Metrics) ExplainFingerprintByQueryID(ctx context.Context, serviceID, qu
 
 	return res, errors.New("query_id doesnt exists")
 }
+
+const selectedQueryMetadataTmpl = `
+SELECT 	any(service_name), any(database), any(schema), any(username), 
+		any(replication_set), any(cluster), any(service_type), any(service_id), 
+		any(environment), any(node_id), any(node_name), any(node_type) 
+FROM 	metrics
+WHERE 	period_start >= :period_start_from 
+	AND period_start <= :period_start_to {{ if not .Totals }} 
+	AND {{ .Group }} = '{{ .DimensionVal }}' {{ end }}
```

Comment:
can you please reformat query, it is very hard to read it with template vars

Reply:
I'm sorry, my bad.
Reformated.